include/net/net_if.h

```diff
@@ -280,13 +280,13 @@ struct net_if_dhcpv4 {
 
 struct net_if_ipv4_autoconf {
 	/** Number of sent probe requests */
-	uint8_t probe_cnt;
+	u8_t probe_cnt;
 
 	/** Number of sent announcements */
-	uint8_t announce_cnt;
+	u8_t announce_cnt;
 
 	/** Incoming conflict count */
-	uint8_t conflict_cnt;
+	u8_t conflict_cnt;
```

Comment:
Actually, there is a hole in the original struct, could you move these 3 attributes to the end of the struct?

Also: the k_delayed_work, could you apply the same as I did for dhcpv4 which you adapted as well in lldp? Better now than later.